### DISCUSSION TITLE
HA dont create sensor

### DISCUSSION THREAD

**Discussion by pidiman** [ESPresense/ESPresense#420] [General]
**Describe the bug**

Hi guys, I’m coming here for advice. I installed espresense on esp32. In HA I see that the sensor is working properly. He shows me the room - workroom. Then I config my Iphone 12 to sensor's configuration file:

```
  - platform: mqtt_room
    name: "FajnyIphone12"
    device_id: "apple:iphone13-2"
    state_topic: "espresense/rooms"
    timeout: 60
    away_time: 120
```
But HA dont create this Sensor.. I see these infos in mqtt explorer:

![Screenshot 2022-03-13 at 19 33 36](https://user-images.githubusercontent.com/25770225/158074080-414247e9-ac97-4f84-bdc6-96da9265c3f4.png)

Any advice ? Thanks a lot!

**Config**

- Version: 2.0.38b7
- Active scan enabled: yes
- Include filter:
- Exclude filter:
- Query filter: 1

**Hardware**

- Firmware flavor: 
- Device (be specific): ESP32-D0WDQ6

**Logs**
```
1647196491: New client connected from 192.168.0.127 as esp32-486f06a4ae30 (p2, c1, k15, u'mqttuser').
1647196499: New connection from 192.168.0.127 on port 1883.
1647196499: Client esp32-486f06a4ae30 already connected, closing old connection.
1647196499: New client connected from 192.168.0.127 as esp32-486f06a4ae30 (p2, c1, k15, u'mqttuser').
1647196507: New connection from 192.168.0.127 on port 1883.
1647196507: Client esp32-486f06a4ae30 already connected, closing old connection.
1647196507: New client connected from 192.168.0.127 as esp32-486f06a4ae30 (p2, c1, k15, u'mqttuser').
1647196515: New connection from 172.30.32.2 on port 1883.
```

**Comment by mehdic** [comment]
Check the entities page. You will find it there.

**Comment by mehdic** [comment]
And update to 2.1.0. It is the most stable version now, as far as I know.

**Comment by pidiman** [comment]
I updated esp to 2.1.0, everything is like before.. 

I see info in MQTT explorer..
I see info in espresense terminal..

But my HA dont create any entities.. 

```
  - platform: mqtt_room
    device_id: "apple:1005:9-26"
    name: 'Apple1005'
    state_topic: 'espresense/rooms'
    timeout: 60
    away_timeout: 120

  - platform: mqtt_room
    device_id: "apple:1007:11-8"
    name: 'Apple1007'
    state_topic: 'espresense/rooms'
    timeout: 60
    away_timeout: 120
```

**Reply by joekeslar** [reply]
Could it be an issue with your use of double and single quotes?   Here's an example from one of my configs, and I can see it in HA just fine...

  - platform: mqtt_room
    device_id: name:bmw_x3
    name: 'ESPresense Car BMW X3'
    state_topic: 'espresense/rooms'
    timeout: 5 
    away_timeout: 0

**Reply by pidiman** [reply]
I tried single and double quotes :(

**Comment by tansuka** [comment]
I have exactly the same issue :(

**Comment by YikLiu93** [comment]
I have the same issue and don't know if there's a workaround?
